draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -324,13 +302,13 @@ Servers should be careful when returning RateLimit header fields in redirection
 ~~~ http-message
 HTTP/1.1 301 Moved Permanently
 Location: /foo/123
-RateLimit: limit=10, remaining=0, reset=10
+RateLimit: problemPolicy;r=0, t=10
 
 ~~~
 
 If a response contains both the Retry-After and the RateLimit header fields, the reset keyword value SHOULD reference the same point in time as the Retry-After field value.
 
-When using a policy involving more than one time window, the server MUST reply with the RateLimit header fields related to the time window with the lower remaining keyword values.
+When using a policy involving more than one time window, the server MUST reply with the RateLimit header field related to the time window with the lower remaining keyword values.
```

Comment:
```suggestion
When using a policy involving more than one time window, the server MUST reply with the RateLimit header field related to the time window with the lower "r" parameter values.
```
Does this need to be updated to be consistent with the change in https://github.com/ietf-wg-httpapi/ratelimit-headers/pull/130/files#diff-d26a6f407efe0e4913cbcf7bf0e2a164bee5edea8bf2fdc4aa97a7ceb3acc03cR229 (s/remaining keyword/r parameter/)?

There are other instances in this document that are still referring to `remaining keyword` that I can't comment on because they're not part of the PR diff - do those need to be updated too?

More generally, I see some diffs where wording was changed from `keyword` to `parameter`, but `keyword` remains in other areas. I'm new to this side of proposal drafts, so apologies if this is just bikeshedding 😁: should this wording be consistent throughout?